src/style.css

```diff
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 900;
+}
+
+body {
+  margin-top: 0;
+}
+
+.form {
+  position: relative;
+}
+
+.form--big {
+  margin: 20px 0;
+}
+
+.icon {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.icon--big {
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
```

Comment:
no need to give the modifier `--big`, just add it for the icon, the modifier is needed only for the small one